### ISSUE TITLE
How do I could get training accuracy?

### ISSUE THREAD

**Issue by SimoCance** [fchollet/keras#5219]
I would to plot training and test accuracy over the epochs. I can get test accuracy _acc_ by:
 `model.test_on_batch(x_test, y_test)`
`model.metrics_names`
but I don't know how to get training accuracy.

**Comment by bstriner** [comment]
Use model.evaluate instead of test_on_batch. That way it will batch your test data instead of trying to run it all at once.

1) Get it as the return value from fit.
2) Call `model.evaluate(x_train, y_train)`.

make sure when you compile to set `metrics=['accuracy']`.

Cheers
